students/ZhuojingXie/lesson04/dict_lab.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+
+
+#Dictionaries 1
+d1 = {
+    'name': 'Chris',
+    'city': 'Seattle',
+    'cake': 'Chocolate'
+}
+print(d1)
+
+d1.pop('cake')
+print(d1)
+
+d1['fruit'] = 'Mango'
+print(d1.keys())
+print(d1.values())
+print('cake' in d1.keys())
```

Comment:
The `in` keyword checks for membership of keys in dict, so this can be `print('cake' in d1)`